articles/introduction.html

```diff
@@ -119,44 +120,44 @@
 </p>
 
 <p>
-Go programs often compile fast.
+<code>gc</code> compiles Go programs very fast.

```

Comment:
Go programs are compiled fast mainly for the syntax design and some line break limitations, not for the effort put in development of "gc". It is more language related than compiler related.

Reply:
We may go back to sth like: "Go compiles and builds programs very fast."

Just IMHO the speed is not only due to intentionally simple design of the language (even so this is a very important aspect to start with), but also due to optimizations in the standard tool chain.

Further - as purely stylistic / aesthetic touch - I thought the repetition of the prev para intro sounds nice:
- <code>gc</code> supports ...
- <code>gc</code> compiles ...

On the other hand, especially also looking at the preceding paragraphs I even feel that we do not need to mention gc so often (which is even some un-word) but just attribute the characteristics to 'Go' ...

Reply:
And on second thought, and as You wish to emphasize the language characteristics, I shall commit "Programs written in go language mostly compile very fast."